src/Compilers/Test/Utilities/VisualBasic/ParserTestUtilities.vb

```diff
@@ -539,38 +541,41 @@ Public Module VerificationHelpers
         End Function
     End Class
 
-    Friend Sub AppendSyntaxErrors(errors As IEnumerable(Of Diagnostic), output As StringBuilder)
+    Friend Sub AppendSyntaxErrors(errors As IEnumerable(Of Diagnostic), output As PooledStringBuilder)
```

Comment:
> Friend Sub AppendSyntaxErrors(errors As IEnumerable(Of Diagnostic), output As PooledStringBuilder) [](start = 3, length = 99)

It doesn't feel like we should be changing this signature. Can't we pass the underlying StringBuilder when we have a PooledStringBuilder?